README.md

```diff
@@ -0,0 +1,12 @@
+# microblog
+
+## API
+Installing API sources in development mode
+```commandline
+pip install -e '.[dev]'
```

Comment:
![image](https://github.com/kozlowskimaciej/microblog/assets/59119141/2f067ceb-cc11-477f-8ea6-d06cfc423acd)

Encountered errors when trying to execute this line

Reply:
Try again but with pulled tags

Reply:
Now it works